modular_skyrat/modules/aesthetics/clothing/clothing.dm

```diff
@@ -1,66 +1,3 @@
-
```

Comment:
Alternatively to all the above review comments, you could keep this file as a means of modularly overwriting `icon` and `worn_icon`, putting your new paths in their place. It'd mean a lot less //SKYRAT EDIT's

Reply:
I again wanna reiterate on this; rather than all the skyrat edits, this may be a much easier way to overwrite them all